### ISSUE TITLE
The problem with repost

### ISSUE THREAD

**Issue by Gorfar** [vdrhtc/concrypt#3]
I don't why, but repost with comment (I've made a repost from the site and add comment) is represented in a strange way. 

See file attached.
![chats2](https://cloud.githubusercontent.com/assets/4509284/11897722/12be3bdc-a5a3-11e5-9a45-54754d65c5a1.png)

**Comment by Gorfar** [comment]
It represents only the comment. Maybe it isn't a problem, just my wish. 
I think, it should be noted that there is a repost. Just mention or repost as itself.

**Comment by vdrhtc** [comment]
Right, reposting is not yet supported, but I'll add this functionality before alpha release.

**Comment by Gorfar** [comment]
I hope.

:D
